NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 * Added add_phylopic_legend (#83)
 * Added permalink generation option to get_attribution (#81)
+* Fix pick_phylopic attribution display bug (#104)
```

Comment:
Should it be reported in NEWS.md if we introduced the bug and fixed the bug between releases?

Reply:
Maybe not, I guess many users don't use the development version?

Reply:
Yeah, I doubt many people do, and I think this might just cause more confusion for those that don't (not that too many people are going to read the changelog anyways).